lib/ansible/modules/cloud/openstack/os_recordset.py

```diff
@@ -173,21 +164,14 @@ def main():
                            supports_check_mode=True,
                            **module_kwargs)
 
-    if not HAS_SHADE:
-        module.fail_json(msg='shade is required for this module')
-    if StrictVersion(shade.__version__) <= StrictVersion('1.8.0'):
-        module.fail_json(msg="To utilize this module, the installed version of "
-                             "the shade library MUST be >1.8.0")
-
     zone = module.params.get('zone')
     name = module.params.get('name')
     state = module.params.get('state')
 
+    shade, cloud = openstack_cloud_from_module(module, min_version='1.8.0')
```

Comment:
The old comparison is doing <=, not just <. So we need min_version='1.9.0' I believe.